applications/FluidDynamicsApplication/tests/TwoFluidInletTest/materials.json

```diff
@@ -20,7 +20,7 @@
                 "name" : "Newtonian2DLaw"
             },
             "Variables"        : {
-                "DENSITY": 1.0E+03,
+                "DENSITY": 1.0E+00,
```

Comment:
no changes in results?

Reply:
Only the level set is checked in this test.